'make' fails with Go versions < 1.8

With go 1.7.x (e.g. 1.7.7)

'make' fails with:

```
Makefile:147: recipe for target '/<gopath>/src/github.com/fabric8-services/fabric8-wit/bin/wit' failed
```

This comes from:
```
# github.com/fabric8-services/fabric8-wit/vendor/github.com/armon/go-metrics
vendor/github.com/armon/go-metrics/inmem_endpoint.go:72: undefined: sort.Slice
vendor/github.com/armon/go-metrics/inmem_endpoint.go:86: undefined: sort.Slice
vendor/github.com/armon/go-metrics/inmem_endpoint.go:113: undefined: sort.Slice
```

sort.Slice is 1.8+. The documentation should be updated to specify requiring go 1.8+ or an older version of armon/go-metrics should be used.

Switching to go 1.8.x (e.g. 1.8.4) resolves the issue.

--- a/README.adoc
+++ b/README.adoc
@@ -19,7 +19,7 @@ link:docs/development/getting-started-win.adoc[Getting started with fabric8-wit 
 
 You need to install:
 
-* `go` (>= v1.7)
+* `go` (>= v1.8)
 * `git`
 * `mercurial`
 * `make`
@@ -32,7 +32,7 @@ Run the following command to find out your Go version.
 $ go version
 ----
 
-*You must at least have Go version 1.7.*
+*You must at least have Go version 1.8.*
 
 See <<fetch-dependencies>> to see an explanaition on how we deal with
 dependencies.

